WTG.Analyzers.TestFramework/Helpers/DiagnosticUtils.cs

```diff
@@ -9,11 +9,6 @@ namespace WTG.Analyzers.TestFramework
 {
 	public static partial class DiagnosticUtils
 	{
-		public static async Task<Diagnostic[]> GetDiagnosticsAsync(DiagnosticAnalyzer analyzer, params string[] sources)
```

Comment:
Dead code?

Reply:
It is now. I moved all the use cases over to use the other overload. This overload makes it too easy to miss out on the language version.